pkg/urbit/noun/manage.c

```diff
@@ -496,7 +496,7 @@ _pave_south(c3_w* mem_w, c3_w siz_w, c3_w len_w)
 static void
 _pave_parts(void)
 {
-  u3R->cax.har_p = u3h_new();
+  u3R->cax.har_p = u3h_new_cache(50000);
```

Comment:
When I was exploring how the cache worked, I had to set this to 800,000 for an urbit to be able to boot. However, I also didn't remove `+play`, etc, and if those are the majority of the cache entries, maybe this is OK.

Reply:
I was able to boot a solid pill with 50k without issue, but I'll check a brass pill.  Uncapped, removing these caches reduced the number of cache entries for compiling hoon.hoon by about 85%.

Another thing I'd like to do is just make this configurable with a CLI flag, so if something goes wrong we have a quick backup.  I know how to add a CLI flag to the daemon, is there an example I can look at of one that gets passed to the worker?

Reply:
All the current arguments are just flags -- they get packed into a bitmask and sent to the worker. But there is one argument with a value on `ipc-redux`, and I just hacked it in, see: dfa818a069dd6ba201b4cfbe2e171275e9f044af.

Reply:
I was able to boot with a brass pill

Reply:
As long as it's not optional, I guess I can just add it as `argv[4]`, and merging that with ipc-redux will just be incrementing a few indices.